tensorflow/core/kernels/conv_ops_fused.cc

```diff
@@ -183,18 +192,24 @@ class FusedResizeAndPadConvFunctor {
                 T1 in_value;
                 if ((conv_in_x >= 0) && (conv_in_x < padded_width) &&
                     (conv_in_y >= 0) && (conv_in_y < padded_height)) {
-                  const T1 top_left(
-                      input_data(batch, top_y_index, left_x_index, in_channel));
-                  const T1 top_right(input_data(batch, top_y_index,
-                                                right_x_index, in_channel));
-                  const T1 bottom_left(input_data(batch, bottom_y_index,
-                                                  left_x_index, in_channel));
-                  const T1 bottom_right(input_data(batch, bottom_y_index,
-                                                   right_x_index, in_channel));
-                  const T1 top = top_left + (top_right - top_left) * x_lerp;
-                  const T1 bottom =
-                      bottom_left + (bottom_right - bottom_left) * x_lerp;
-                  in_value = top + (bottom - top) * y_lerp;
+                  if (SampleMode == NEAREST) {
+                    const T1 top_left(input_data(batch, top_y_index,
```

Comment:
NEAREST doesn't seem like the right name here, since it always takes the top-left one.

perhaps comment (or check) somewhere that DoResize and SampleMode need to be in sync?  Seems like using DoResize==true with SampleMode=NEAREST would be wrong.   Or maybe just use DoResize for this choice here, and remove SampleMode for now?